src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseCommentAttributes.java

```diff
@@ -207,6 +208,15 @@ public int compare(FeedbackResponseCommentAttributes frc1, FeedbackResponseComme
         });
     }
 
+    public static void sortFeedbackResponseCommentsByCreationTime(FeedbackResponseCommentAttributes...frcsToSort) {
+        Arrays.sort(frcsToSort, new Comparator<FeedbackResponseCommentAttributes>() {
+            @Override
+            public int compare(FeedbackResponseCommentAttributes frc1, FeedbackResponseCommentAttributes frc2) {
+                return frc1.createdAt.compareTo(frc2.createdAt);
+            }
+        });
+    }
```

Comment:
BTW, there two sort methods are very similar. Try to reduce duplication if possible.